templates/common/openstack/files/ipv6-config.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .IPFamilies "DualStack"}}
```

Comment:
Note for future patches: for code that is supposed to go away, I'd really like to see a comment explaining why it's there with a link to an open issue.

Reply:
Worth noting that there is also a DualStackIPv6Primary ip family setting now too that should probably be included in this conditional. I don't know if it's a major concern for you right now, but for customers who want dual stack with an eye toward moving to single stack v6 they'll need to use that.